server/queries/queries.js

```diff
@@ -0,0 +1,104 @@
+const { Client } = require('pg');
+const Promise = require('promise');
+
+let pool;
+
+if (process.env.DATABASE_URL) {
+    pool = new Client({
+        connectionString: process.env.DATABASE_URL
+    });
+} else {
+    pool = new Client({
+        user: process.env.USER,
+        host: process.env.HOST,
+        database: 'telltales',
+        password: process.env.PASSWORD,
+        port: process.env.PORT
+    });
+}
+
+pool.connect((err) => {
+    if (err) {
+        console.error('Connection error', err.stack);
+    } else {
+        console.log("Connected");
+    }
+});
+
+// Create the table if it doesn't already exist
+/* eslint max-len: ["error", { "code": 200 }] */
+const createTableQuery = 'CREATE TABLE IF NOT EXISTS users ( id SERIAL PRIMARY KEY, name VARCHAR(15) NOT NULL, score INT );'
```

Comment:
I think there's a more declarative way to do this.
I think Tilly declared all the domain entities and did `sequelize.sync()`. it was easier to guarantee idempotence and support incremental changes.

Reply:
Should I change the whole file to use Sequelize instead of Pool then?